setup.py

```diff
@@ -49,16 +55,7 @@ def main():
                      glob(os.path.join('civis', 'tests', '*.json')))],
         long_description=README,
         long_description_content_type="text/x-rst",
-        install_requires=[
-            'pyyaml>=3.0,<=5.99',
-            'click>=6.0,<=7.99',
-            'jsonref>=0.1.0,<=0.2.99',
-            'requests>=2.12.0,==2.*',
-            'jsonschema>=2.5.1,<=3.99',
-            'joblib>=0.11,<2',
-            'cloudpickle>=0.2.0,<2',
-            'tenacity>=6.2,<7',
-        ],
+        install_requires=requirements,
```

Comment:
I was debating if we should specify dependencies and their versions here in `setup.py` only or in `requirements.txt` only (that's what #190 is about). I've decided to keep `requirements.txt`, since the Python packaging world is very much still in flux. With all the emerging, competing tools (pipenv, Poetry, pip-tools, etc.) and standards (Pipfile, Pipfile.lock, pyproject.toml, etc.), there seems to be a slight preference for having a `requirements.txt` file laying around, since its content is more introspectable than what's embedded in `setup.py`. Though honestly, I don't know when, if ever, we'll move to one of these standards/tools.